library/alloc/src/vec/mod.rs

```diff
@@ -3025,26 +3025,29 @@ impl<T: Clone, A: Allocator> Vec<T, A> {
         self.spec_extend(other.iter())
     }
 
-    /// Copies elements from `src` range to the end of the vector.
+    /// Copies elements from one part of the vector to the end.
     ///
-    /// # Panics
+    /// `src` is the range that the elements will be copied from.
```

Comment:
Reading #104762, I think an elaboration or rephrasing of docs is in order. I'd maybe word it as:

```rust
/// Given a range `src`, clone a slice of elements in that range and append it to the end.
/// `src` must be a valid range where it can form a valid subslice of the `Vec`.
```

Reply:
"Copy" is probably a bad term since the impl block only requires `T: Clone`.